src/Reducer/tokenClient.js

```diff
@@ -0,0 +1,12 @@
+const initialState = {
+  tokenClient: "0",
+};
+
+export function tokenClientReducer(state = initialState, action) {
+  switch (action.type) {
+    case "SET_TOKEN":
+      return { ...state, tokenClient: action.test };
```

Comment:
It'd be better to give explicit names to your keys, because `action.test` stored in `state.tokenClient` doesn't really make any sense